src/main/java/net/minecraftforge/client/model/pipeline/VertexLighterFlat.java

```diff
@@ -210,19 +210,18 @@ protected void processQuad()
                         pos[2] += blockInfo.getBlockPos().getZ();*/
                         parent.put(e, position[v]);
                         break;
-                    case NORMAL: if(normalIndex != -1)
```

Comment:
Why was this logic changed?

Reply:
That check does nothing (other than be confusing). `normalIndex` should never be set to -1, and if it ever was, the code above would crash anyway.

As this ended up moving slightly from my original intent of fixing vertex format switch blocks, I can just leave this out, but it should probably be fixed at some point - keeping it like this just leads to misunderstanding.